modules/webadmin.cpp

```diff
@@ -210,6 +210,13 @@ class CWebAdminMod : public CModule {
             pNewUser->SetPass(sHash, CUser::HASH_DEFAULT, sSalt);
         }
 
+        sArg = WebSock.GetParam("builtinauthdisabled");
+        if(spSession->IsAdmin()) {
+            if(!sArg.empty()) {
+                pNewUser->SetBuiltinAuthDisabled(sArg.ToBool());
+            }
+        }
```

Comment:
Doesn't this need `else if (pUser) { pNewUser->SetBuildinAuthDisabled(pUser->BuiltinAuthDisabled()); }`? When a user is edited, instead of resetting this setting to false, it should be preserved. Right?
(I'm not sure if the above is necessary, but e.g. BindHost is handled in a similar way below)

Reply:
Yeah, I think you're right. I didn't run into this while testing but will push something just in case. If it ends up being unnecessary I'll go ahead and revert.